_posts/2018-04-25-productivity-with-scripts-screencapture.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "Increasing Productivity with Scripts: Utilizing Mac's screencapture Utility"
+date:   2018-04-25 12:00:00 -0400
+categories: automation scripts
+author: Scott Davidson
+---
+
+I am a firm believer that there is almost always room for improvement in process, especially when it comes to automating tasks. As an Automation Engineer here at Forward Financing, I am always trying to ensure optimal productivity for my team. However, I am also constantly looking for improvements that I can make within my personal process to help me save time throughout the day.
```

Comment:
Lots of "I am"s in this first paragraph. You can eliminate the present continuous tense for a more affirmative voice, ie: I try, I look for improvements, etc.